lnrpc/routerrpc/router_backend.go

```diff
@@ -142,37 +142,41 @@ func (r *RouterBackend) QueryRoutes(ctx context.Context,
 		ignoredNodes[ignoreVertex] = struct{}{}
 	}
 
-	ignoredEdges := make(map[routing.EdgeLocator]struct{})
+	ignoredEdges := make(map[routing.DirectedNodePair]struct{})
 	for _, ignoredEdge := range in.IgnoredEdges {
```

Comment:
Should we change the `RPC` param to use node pairs instead of chans also?

Reply:
RPC parameter added and `ignored_edges` deprecated